fedorchenkory/second/task 1.py

```diff
@@ -0,0 +1,22 @@
+"""
+Користувач воодить речення на англійській мові, розділяючи слова пробілами. Вивести список, кожний елемент якого є списком, де
+перший елемент - кожне друге слово речення, а другий кількість повторів цьогого слова у реченні.
+"""
+import re
+def find_text(text):
+    n= re.split(' ',text)
```

Comment:
Nice, and some mentions for you.
You can use fin = re.findall('\w+', text) here instead of n= re.split(' ',text), so that all for-loop above could be removed
`    fin=[]
    for i in n:
        b=re.findall('[A-Za-z]{1,}',i)
        if b =='':
            b=' '
        fin.append(b[0])
        print(b)`